src/checkout/utils/contact.ts

```diff
@@ -0,0 +1,21 @@
+interface AlternateSubdivisions {
+  intlSubdivision?: string
+  usSubdivision?: string
+}
+
+interface FinalSubdivision {
+  subdivision: string
+}
+
+interface ContactBase {
+  street1: string
+  street2: string
+  city: string
+  country: string
+  postalCode: string
+}
+
+export type Contact = ContactBase & AlternateSubdivisions
+export type BackendContact = ContactBase & FinalSubdivision
+
+export const isUS = (country: string): boolean => country === 'United States'
```

Comment:
are any of these being used?

Reply:
yes, they are used. `isUS` is used in the checkout forms.

Reply:
Removed what wasn't used.